### ISSUE TITLE
About update

### ISSUE THREAD

**Issue by xianling88** [sayem314/hooman#34]
#### What would you like to discuss?

> Hello, really cool stuff here, but since the cloudflare updated their method, this cant work anymore,
Is there any chance you will update in the near future??? thx a lot man, u have a good day.

#### Checklist

- [x] I have read the hooman and got documentation.

**Comment by sayem314** [comment]
Yes soon! I plan to fix it within the next weeks :) Atm I'm very busy with other internal projects.

**Comment by xianling88** [comment]
Awesome man, looking forward to it.

**Comment by chaoticvibe** [comment]
Some time ago, about two months ago, it works perfect in a website (protected by cloudflare), but now days it's not working, i hope a update that fix it soon, this is a amazing tool, congratulations!

**Comment by x6pnda** [comment]
ye, looking around of node libraries to bypass these stupid cloudflare screens. Hopefully the creator picks back up the project soon :)

**Comment by andress134** [comment]
> Yes soon! I plan to fix it within the next weeks :) Atm I'm very busy with other internal projects.

U will still offer updates for this lib? im checking and can't bypass v1 challenge (iuam)

**Comment by sayem314** [comment]
I’m really very sorry for not updating as promised. I got really busy with big private projects that I cannot make any time for working on my active open-source projects.

**Comment by AlfredoDaAs** [comment]
By reading the code and testing the body response, the new version never gets inside the 503 status code statement (hooman.js file, line: 56 - response.body.includes('var s') ), it doesn't find a 'var s' in the body. Also inside core.js which solves the challenge, it's looking for a 'var s' in the script string... I'm new with this cloudflare stuff, but this is not making sense to me, is the body response different than before?

anyways... the library is amazing, never seen something like this before.
